### ISSUE TITLE
Issue with 'Sign in' button in the login method

### ISSUE THREAD

**Issue by leoarios27** [jg-fisher/LinkedinBot#1]
I used the code as intended, but an error popped up when trying to find/click the submit button on the LinkedIn login page (screenshot attached below). I searched for a different piece of code to replace the error line and found '//*[@type = "submit"]' worked for me. There are other solutions to this problem (linked below).

I am at a relatively beginner level of coding in python so it could be a mistake on my end, but hopefully this helps someone should they run into the same problem

https://www.linkedin.com/pulse/how-easy-scraping-data-from-linkedin-profiles-david-craven

![linkedinLoginError_scrubbed](https://user-images.githubusercontent.com/22604255/94954243-d93acb00-04b6-11eb-8b64-e1bee85d8ffb.png)

**Comment by Pr0digyu** [comment]
> I used the code as intended, but an error popped up when trying to find/click the submit button on the LinkedIn login page (screenshot attached below). I searched for a different piece of code to replace the error line and found '//*[@type = "submit"]' worked for me. There are other solutions to this problem (linked below).
> 
> I am at a relatively beginner level of coding in python so it could be a mistake on my end, but hopefully this helps someone should they run into the same problem
> 
> https://www.linkedin.com/pulse/how-easy-scraping-data-from-linkedin-profiles-david-craven
> 
> ![linkedinLoginError_scrubbed](https://user-images.githubusercontent.com/22604255/94954243-d93acb00-04b6-11eb-8b64-e1bee85d8ffb.png)

Yes, the name of the class probably changed. Try the following code for login:
    def login(self, username, password):
        """ Login to LinkedIn account """
        self._nav(self.login_url)
        self.driver.find_element_by_id('username').send_keys(self.username)
        pw = self.driver.find_element_by_id('password')
        pw.send_keys(self.password)
        pw.send_keys(Keys.ENTER)
